trulens_eval/trulens_eval/feedback/provider/base.py

```diff
@@ -142,11 +142,17 @@ def _extract_score_and_reasons_from_response(
         response = self.endpoint.run_me(
             lambda: self._create_chat_completion(messages=llm_messages)
         )
-        if "Supporting Evidence" in response:
+        if "Supporting Evidence" or "Reason" in response:
```

Comment:
are we still preserving the "Supporting Evidence" keyword in COT prompt - seems like we now inject it in summarization & groundedness but no longer in COT prompt? 

if so, could you update the docstring above in `_extract_score_and_reasons_from_response`?